### PR TITLE
Update license to mention OpenRefine contributors

### PR DESCRIPTION
**Pull request by wetneb**
At the moment, the copyright notice only mentions Google Inc., which feels weird given that a lot of work has been done since, so I feel like it's worth adding a mention of "OpenRefine contributors", which has been the way we credit newer contributions in copyright headers throughout the project.

This is motivated by the license being shown in the Windows installer, to be released for the first time in 3.8.

image

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Copyright 2010, Google Inc. All rights reserved.
+Copyright 2024, OpenRefine contributors.
```
Comment (tfmorris) [inline LICENSE.txt:2]:
```suggestion
Copyright 2010, 2024 Google Inc. and OpenRefine contributors. All rights reserved.
```

Comment (tfmorris) [reply LICENSE.txt:2]: I started contributing in 2010, so I don't think we need to use a separate, later date.

**Review by tfmorris — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (wetneb) [reply LICENSE.txt:2]: Fine by me too!

File: LICENSE.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright 2010, Google Inc. All rights reserved.
+Copyright 2010, 2024 Google Inc. and OpenRefine contributors. All rights reserved.
```
[the remark this answers is not in the document]
Comment (wetneb) [reply LICENSE.txt:1]: Yes, I'd be in favour of removing years too, but then technically that's altering the original copyright statement, which is against the license, perhaps…
Maybe we go back to separate lines, keeping the year for Google (leaving that statement untouched) and without year for OpenRefine contributors? Then we don't feel the need to bump a number each year (which feels a bit futile).

**Comment by magdmartin** [comment]
I reached out to CS&S for confirmation

**Comment by magdmartin** [comment]
Here is the answer from @joehand 
> Adding the date and contributors seems like a good path: Copyright 2012 OpenRefine contributors. And link to the original Google groups discussion in the PR. https://groups.google.com/g/openrefine/c/a3R6afKb4-4

I am attaching the full conversation for reference. 
[Mail - Request for Feedback_ Proposed Amendments to OpenRefine Copyright Information.pdf](https://github.com/OpenRefine/OpenRefine/files/14338469/Mail.-.Request.for.Feedback_.Proposed.Amendments.to.OpenRefine.Copyright.Information.pdf)

**Review by magdmartin — APPROVED** [review]

File: LICENSE.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Copyright 2010, Google Inc. All rights reserved.
-Copyright OpenRefine contributors. All rights reserved.
+Copyright 2012, OpenRefine contributors.
```
Comment (tfmorris) [inline LICENSE.txt:2]:
```suggestion
Copyright 2010, OpenRefine contributors.
```

As I mentioned earlier, I started contributing in 2010. 9c6097a06560fabc8a9ba2c45a23522911cb425f

**Review by tfmorris — CHANGES_REQUESTED** [review]

**Review by tfmorris — APPROVED** [review]

**Pull request merged**